engine/src/main/java/com/datatorrent/stram/codec/LogicalPlanSerializer.java

```diff
@@ -203,6 +221,201 @@ public boolean useForType(JavaType t)
     return result;
   }
 
+  /**
+   * @param flatten
+   * @param dag
+   * @return
+   */
+  public static Map<String, Object> convertToMap(LogicalPlan dag, boolean flatten)
```

Comment:
With addition of modules there are 2 scenarios.
1. Showing logical plan with modules as logical enitity 
2. Flattening the logical plan i.e modules and operators within modules are shown.
This is the reason 2nd argument is passed .
Also i wasn't sure whether i should change the existing methods signature ( backward compatibility ) hence added a new one.